pkg/handlers/list_files_test.go

```diff
@@ -128,7 +129,7 @@ func TestListFilesHandler_ServeHTTP(t *testing.T) {
 			router.ServeHTTP(rr, req)
 
 			assert.Equal(t, tt.wantStatusCode, rr.Code)
-			assert.Contains(t, rr.Body.String(), tt.wantBody)
+			assert.Equal(t, strings.TrimSuffix(rr.Body.String(), "\n"), tt.wantBody)
```

Comment:
`assert.Equal` expected the first argument to be an expected value, it's important for proper error messaging:
`func assert.Equal(t assert.TestingT, expected interface{}, actual interface{}, msgAndArgs ...interface{}) bool`